openreview/venue_request/venue_request.py

```diff
@@ -1423,17 +1423,23 @@ def setup_request_form(self):
                 'default': '2',
                 'order': 41
             },
+            'submission_license': {
```

Comment:
how does organizer specify that the submission license should be a dropdown/checkbox/radio instead of string?